tools/translation/generate_pot.sh

```diff
@@ -24,49 +24,64 @@
 #
 # ===========================================================================
 
-set -e # exit if a command fails
+set -u -e -o pipefail
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 repo_dir="$(realpath "${script_dir}/../..")"
 dpk_dir="${repo_dir}/pkg/unvanquished_src.dpkdir"
 pot_dir="${dpk_dir}/translation"
-main_pot_file="${pot_dir}/game.pot"
-commands_pot_file="${pot_dir}/commands.pot"
 
-temp_pot_file="$(mktemp)"
+. "${script_dir}/translation.conf"
```

Comment:
sets basic config vars

Reply:
Is it the same as `source`? If so please use that instead of cryptically golfing

Reply:
Yes, it's the same as source. Technically speaking, source is a bash-ism and `. foo` is more portable, but we do enforce bash anyways so its probably fine.

Reply:
Yes, this is the standard syntax for including a file. I an use `source` indeed since there is no plan to make it compatible with another language than bash.

Reply:
TIL, `.` is OK I guess